cmd/analyze/main.go

```diff
@@ -61,29 +61,14 @@ func main() {
 		log.Label("localPath", *localPkg),
 		log.Label("version", *version))
 
-	var pkg *pkgecosystem.Pkg
-	var err error
-	if *localPkg != "" {
-		if *version != "" {
-			log.Panic("Unable to specify version for local packages")
-		}
-		pkg = manager.Local(*pkgName, *version, *localPkg)
-	} else if *version != "" {
-		pkg = manager.Package(*pkgName, *version)
-	} else {
-		pkg, err = manager.Latest(*pkgName)
-		if err != nil {
-			log.Panic("Failed to get latest version",
-				log.Label("ecosystem", *ecosystem),
-				log.Label("name", *pkgName))
-		}
+	pkg, err := manager.ResolvePackage(*pkgName, *version, *localPkg)
+	if err != nil {
+		log.Panic("Error resolving package",
+			log.Label("ecosystem", *ecosystem),
+			log.Label("name", *pkgName),
+			"error", err)
 	}
 
-	log.Info("Got request",
```

Comment:
We lose this log message. Is that intentional?

Reply:
Yes. It actually gets logged above as well (it was being logged twice).

Reply:
ack!